src/bootstrap.rs

```diff
@@ -1,24 +1,33 @@
+/*!
+Utilities for bootstrapping an app that uses the Windows App SDK.
+!*/
 use bindings::Windows::Win32::{
     Foundation::HWND,
     UI::WindowsAndMessaging::{MessageBoxW, MB_ICONERROR, MB_OK},
 };
 
+/// A target version of the Windows App SDK.
```

Comment:
Let's skip documenting this structure as I missed making it private in the last pass and it will be replaced soon.

Context: It's currently used to represent the minimum framework package that's compatible with the app -and- the exact version of the dynamic dependency lifetime manager, a separate component. It's a bit confusing, I know. How this will all _work_ is an on-going discussion with Microsoft. 😅

Reply:
I'll leave the documentation since it's probably not harmful to mention what it is. I'll make it private though so it doesn't show in the generated docs.

Reply:
Good idea!